databases/week2/more-queries.sql

```diff
@@ -0,0 +1,24 @@
+USE hyf;
+select * from task;
+
+-- 1 
+SELECT  user.name, user.email FROM user
```

Comment:
you need to select all the tasks that are assigned to users whose email ends in `@spotify`, that means you need to join the tables task and user ;)